client/js/renderPreview.js

```diff
@@ -12,6 +12,19 @@ function renderPreview(preview, msg) {
 		return;
 	}
 
+	preview.link = Handlebars.Utils.escapeExpression(preview.link);
```

Comment:
On `master`:

<img width="689" alt="screen shot 2017-07-21 at 12 01 47" src="https://user-images.githubusercontent.com/113730/28471840-75cc98b2-6e0c-11e7-812f-538c29e34e5b.png">

In this PR:

<img width="671" alt="screen shot 2017-07-21 at 12 02 45" src="https://user-images.githubusercontent.com/113730/28471866-8875c894-6e0c-11e7-93a9-86b514b24722.png">

:man_shrugging: